viz/index.js

```diff
@@ -1,142 +1,171 @@
+  else if (city == "dc") cityName = "Washington DC";
+  else if (city == "chicago") cityName = "Chicago";
+  else if (city == "nyc") cityName = "New York City";
+  document.getElementById("city").innerHTML = cityName;
+};
 
 const setupYearButtons = (yearsList) => {
-  let yearSelector = document.getElementById('year-buttons-container');
+  let yearSelector = document.getElementById("year-buttons-container");
   // clear out old buttons
   yearSelector.innerHTML = null;
   yearsList.forEach((year) => {
-    let button = document.createElement('button');
-    button.setAttribute('id', 'year-button-' + year.toString());
+    let button = document.createElement("button");
+    button.setAttribute("id", "year-button-" + year.toString());
     button.innerHTML = year.toString();
-    button.addEventListener('click', () => selectYear(year));
+    button.addEventListener("click", () => selectYear(year));
     yearSelector.appendChild(button);
   });
-}
+};
 
 const selectYear = (year) => {
   year = parseInt(year);
-  if (year == currentYear && incomeLayerGroups.length)
-    return;
+  if (year == currentYear && incomeLayerGroups.length) return;
   let previousYear = currentYear;
   currentYear = year;
+  console.log(year);
   redrawMapLayers(previousYear, currentYear);
   updateYearHTML(year);
   updateURLParams();
 };
 
 const updateYearHTML = (year) => {
-  document.querySelectorAll('button').forEach((btn) => {
-    btn.className = '';
-    if (btn.id === 'year-button-' + year.toString()) {
-      btn.className = 'active';
+  document.querySelectorAll("button").forEach((btn) => {
+    btn.className = "";
+    if (btn.id === "year-button-" + year.toString()) {
+      btn.className = "active";
     }
   });
   yearElt.innerHTML = year.toString();
-}
-
+};
 
 const toggleBikeStationLayer = () => {
   updateURLParams();
   redrawMapLayers(currentYear, currentYear);
-}
+};
 const toggleRaceLayer = () => {
   updateURLParams();
   redrawMapLayers(currentYear, currentYear);
-}
+};
 const toggleIncomeLayer = () => {
   updateURLParams();
   redrawMapLayers(currentYear, currentYear);
-}
+};
 
-const bikeCheck = document.getElementById('b-bikes');
-const incomeCheck = document.getElementById('b-income');
-const raceCheck = document.getElementById('b-race');
-bikeCheck.addEventListener('click', toggleBikeStationLayer);
-incomeCheck.addEventListener('click', toggleIncomeLayer);
-raceCheck.addEventListener('click', toggleRaceLayer);
+const toggleBorderLayer = () => {
+  updateURLParams();
+  redrawMapLayers(currentYear, currentYear);
+};
 
+const bikeCheck = document.getElementById("b-bikes");
+const incomeCheck = document.getElementById("b-income");
+const raceCheck = document.getElementById("b-race");
+let borderCheck = false;
+bikeCheck.addEventListener("click", toggleBikeStationLayer);
+incomeCheck.addEventListener("click", toggleIncomeLayer);
+raceCheck.addEventListener("click", toggleRaceLayer);
+// borderCheck.addEventListener("click", toggleBorderLayer);
```

Comment:
remove commented out code or add a comment about why it is commented out for now